### PR TITLE
[ONNX] Assert all op types supported

### PR DESCRIPTION
**Pull request by postrational**
Verify that import of all op types is supported and display a list of unsupported ops found in model if any.

### CODE REVIEW

File: src/ngraph/frontend/onnx_import/core/model.cpp [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        Model::Model(const onnx::ModelProto& model_proto)
+            : m_model_proto{&model_proto}
+        {
+            // Verify that the ONNX graph contains only nodes of supported op_type
+            assert_all_op_types_supported();
+        }
+
+        void Model::assert_all_op_types_supported()
+        {
+            std::set<std::string> unsupported_ops;
```
Comment (arogowie-intel) [inline src/ngraph/frontend/onnx_import/core/model.cpp]: Please add `<set>` header file.

File: src/ngraph/frontend/onnx_import/ops_bridge.cpp [hunk 2]
```diff
@@ -109,6 +109,11 @@ namespace ngraph
                     return ops_bridge::get()(node);
                 }
 
+                static bool is_op_type_supported(const std::string& op_type)
```
Comment (arogowie-intel) [inline src/ngraph/frontend/onnx_import/ops_bridge.cpp]: Please add `<string>` header file.

Comment (arogowie-intel) [inline src/ngraph/frontend/onnx_import/ops_bridge.cpp]: This function may also be `const` qualified.

Comment (postrational) [reply src/ngraph/frontend/onnx_import/ops_bridge.cpp]: Static member function cannot have `const` qualifier.

Comment (arogowie-intel) [reply src/ngraph/frontend/onnx_import/ops_bridge.cpp]: Good point! :)

**Pull request merged**